internal/vds/vds.h

```diff
@@ -65,13 +64,13 @@ struct vdsbuffer fence(
     enum interpolation_method interpolation_method
 );
 
-struct vdsbuffer fence_metadata(
+struct response fence_metadata(
     const char* vds,
     const char* credentials,
     size_t npoints
 );
 
-void vdsbuffer_delete(struct vdsbuffer*);
+void response_delete(struct response*);
```

Comment:
would it make more sense to rename this thing into delete_response?

Reply:
I do not agree 😄 Keep in mind this is a C API. It's a very common pattern in C to have some struct / type and a set of function's to operate on the type (since no classes and whatnot). And when you API have multiple types, you get better grouping of operations (functions) by prefixing the type, not the operation (`type_operation`).

Now we don't have a large C API, not a public one. But my point is that `type_operation` is less supprising, or more common if you will, than `operation_type`. I can live with both though.

Reply:
Now you lie in the commit message! You do more than renaming vds buffer to response!
So maybe either make it a separate commit, or listen and agree with Erlend and leave it as is.

Reply:
I guess I am not completely awake yet. I think I would drop this change for now. If it bothers us a lot in the future, we can still change it then.